dotnet/samples/KernelSyntaxExamples/Example57_FunctionEventHandlers.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Events;
+using RepoUtils;
+
+#pragma warning disable RCS1214 // Unnecessary interpolated string.
```

Comment:
Is this needed? I don't see any issues (also they should be easy to resolve, yeah?)